app/src/main/java/com/eveningoutpost/dexdrip/FakeNumbers.java

```diff
@@ -44,5 +46,24 @@ public void onClick(View v) {
             }
 
         });
+        
+        button = (Button)findViewById(R.id.StartTest);
+        button.setOnClickListener(new View.OnClickListener() {
+            public void onClick(View v) {
+                ActiveBgAlert aba = ActiveBgAlert.getOnly();
+                ActiveBgAlert.ClearData();
+                ActiveBgAlert.Create("some string", true, new Date().getTime());
```

Comment:
The file (from a fresh checkout) is not having the "import java.util.Date;". But you could use  System.currentTimeMillis() since it should be the same as new Date().getTime().